extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -158,7 +170,13 @@ export class Bind {
     user().assert(this.enabled_, `Experiment "${TAG}" is disabled.`);
 
     // TODO(choumx): What if `state` contains references to globals?
-    Object.assign(this.scope_, state);
+    if (Object.keys(this.scope_).length > 0) {
```

Comment:
@choumx and I are discussing this here: https://github.com/ampproject/amphtml/pull/8458#discussion_r108539534